### PR TITLE
fix(route): fix ccreports route is empty and refactor to V2

### LINKED ISSUE #8989: 消费者报道原始网址改版导致RSS爬取无效
### 路由地址

 https://rsshub.app/ccreports

### 完整路由地址

https://rsshub.app/ccreports/article

### 相关文档

https://docs.rsshub.app/shopping.html#xiao-fei-zhe-bao-dao

### 预期是什么？

正常抓取RSS

### 实际发生了什么？

消费者报道改版导致爬取错误

### 部署

自建

### 部署相关信息

OS:debian Node version: v14.19.0

### 额外信息

```shell
Error message: this route is empty, please check the original site
```

### 这不是重复的 issue

- [X] 我已经搜索了[现有 issue](https://github.com/DIYgod/RSSHub/issues)，以确保该错误尚未被报告。

### PR DESCRIPTION
**Pull request by Fatpandac**
## 该 PR 相关 Issue / Involved issue

Close #8989

## 完整路由地址 / Example for the proposed route(s)

```routes
/ccreports/article
```

## 新RSS检查列表 / New RSS Script Checklist
  
- [ ] New Route
- [X] Documentation
  - [X] CN
  - [ ] EN
- [X] 全文获取 fulltext
  - [X] Use Cache
- [ ] 反爬/频率限制 anti-bot or rate limit?
  - [ ] 如果有, 是否有对应的措施? If yes, do your code reflect this sign?
- [X] 日期和时间 date and time
  - [X] 可以解析 Parsed
  - [ ] 时区调整 Correct TimeZone
- [ ] 添加了新的包 New package added 
- [ ] `Puppeteer`

### CODE REVIEW

File: lib/v2/ccreports/index.js [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+    const list = $('div.index-four-content > div.article-box')
+        .find('div.new-child')
+        .map((_, item) => ({
+            title: $(item).find('p.new-title').text(),
+            link: new URL($(item).find('a').attr('href'), rootUrl).href,
+            author: $(item)
+                .find('p.new-desc')
+                .text()
+                .match(/作者：(.*?)\s/)[1],
+            pubDate: parseDate(
```
Comment (TonyRL) [inline lib/v2/ccreports/index.js:19]: Please parse the pubDate from the article. It has a full date and time.

**Review by TonyRL — CHANGES_REQUESTED** [review]

File: lib/v2/ccreports/index.js (hunk 1 above)
Comment (Fatpandac) [reply lib/v2/ccreports/index.js]: OK

File: lib/v2/ccreports/index.js [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+        }))
+        .get();
+
+    const items = await Promise.all(
+        list.map((item) =>
+            ctx.cache.tryGet(item.link, async () => {
+                const detailData = await got.get(item.link);
+                const $ = cheerio.load(detailData.data);
+                item.description = $('div.pdbox').html();
+                item.pubDate = parseDate($('div.newbox > div.newtit > p').text(), 'YYYY-MM-DD HH:mm:ss');
```
Comment (TonyRL) [inline lib/v2/ccreports/index.js:28]: You need to adjust the timezone also. See Vercel's output vs the website.

**Review by TonyRL — CHANGES_REQUESTED** [review]

File: lib/v2/ccreports/index.js (hunk 2 above)
Comment (Fatpandac) [reply lib/v2/ccreports/index.js]: Done.

**Pull request merged**